install.sh

```diff
@@ -7,43 +7,22 @@ echo "# This script installs the dependencies for emirge. #"
 echo "#####################################################"
 echo
 
-if [[ ! -f meshmode/setup.py ]]; then
-	echo "ERROR: incomplete git clone. Please run:"
-	echo "  git submodule init && git submodule update"
-	echo "to fetch emirge's submodules."
-	exit 1
-fi
+export MY_CONDA_DIR=$HOME/miniforge3
 
-myos=$(uname)
-[[ $myos == "Darwin" ]] && myos="MacOSX"
+./install-conda.sh
 
-myarch=$(uname -m)
+export PATH=$MY_CONDA_DIR/bin:$PATH
 
-if [[ ! -d ~/miniforge3 ]]; then
-    echo "Installing Miniforge"
-    wget -c --quiet https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-$myos-$myarch.sh
-    bash Miniforge3-$myos-$myarch.sh -b
-fi
+echo "==== Create 'dgfem' conda environment"
+conda init
+conda create --name dgfem --yes
 
-export CONDA=$HOME/miniforge3
-export PATH=$HOME/miniforge3/bin:$PATH
-export OCL_ICD_VENDORS=$HOME/miniforge3/etc/OpenCL/vendors/
-echo 'export OCL_ICD_VENDORS=$HOME/miniforge3/etc/OpenCL/vendors/' >> $HOME/.bashrc
+$MY_CONDA_DIR/bin/activate dgfem
```

Comment:
Did `conda activate dgfem` give you an error here? If so, I think you can add `eval "$(conda shell.bash hook)"` before it.

Reply:
Was thinking you might be able to get rid of the `MY_CONDA_DIR` definition, but it looks like you need it for the `PATH` stuff anyway.